docs/en/latest/tutorials/add-multiple-api-versions.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Add multiple API versions
+keywords:
+  - API Versioning
+  - Multiple APIs
+  - Proxy rewrite
+  - Request redirect
+  - Route API requests
+description: In this tutorial, you will learn how to _publish and manage multiple versions of your API_ with Apache APISIX.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## What is API versioning?
+
+**API versioning** is the practice of managing changes to an API and ensuring that these changes are made without disrupting clients. A good API versioning strategy clearly communicates the changes made and allows API consumers to decide when to upgrade to the latest version at their own pace.
+
+## Types of API Versioning
```

Comment:
Please use sentence case for all headings: https://apisix.apache.org/docs/general/documentation-style-guide/#formatting-punctuation-and-organization